t/02-regression_tests.t

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/perl
+# This program is open source, licensed under the PostgreSQL License.
+# For license terms, see the LICENSE file.
+#
+# Copyright (C) 2012-2022: Open PostgreSQL Monitoring Development Group
+
+# This file gather various regression tests against the same cluster to avoid
+# creating one cluster per regression test.
+
+use strict;
+use warnings;
+
+use Storable ('store');
+use lib 't/lib';
+use pgNode;
+use TestLib ();
+use Test::More tests => 2;
+
+my $node = pgNode->get_new_node('prod');
+
+$node->init;
+
+$node->append_conf('postgresql.conf', 'stats_block_level = on')
+    if $node->version < 8.3;
+
+$node->start;
+
+### Begin of tests ###
+
+# == Regression test for #326 ==
+# check_pga should not complain when using an existing status file missing
+# without an existing lock file.
+my $pga_data = "$TestLib::tmp_check/tmp-status-file.data";
+
+# Create an empty status file
+store( {}, $pga_data);
+# make there's no lock file leftovers from previous tests...
+unlink "${pga_data}.lock";
```

Comment:
I was thinking it would be better to test the other way:
- test that no status file exists
- run a test that relies on the status file
- test that both the expectedstatus file and lock file exist
- remove the lock file
- run the test again, which should work
- possibly test again that the lock file exists

Reply:
I first wrote the tests with two different call to check_pga, to create the status file, remove the lock, and re-run the test again. 

But I finally choose the fast lane as I was wondering what this was buying us in the end of the day? Just a test to make sure check_pga is creating its status file when missing (your second step)?

Reply:
It's just to be thorough, and validate that the 2 possible scenario (totally empty environment and environment upgraded from a previous version) work as intended.  Even if it's really unlikely that it ever gets broken it doesn't cost much to do so.

Reply:
done